fuel-poa-coordinator/README.md

```diff
@@ -0,0 +1,3 @@
+# Fuel Core PoA Manager
+
+Coordinates block production in a PoA network. This is a stopgap implementation until `fuel-bft` can be used, i.e. when PoS network is usable.
```

Comment:
Do we want PoS references in READMEs?

Reply:
There could also be scenarios where a PoA configuration is desired in the future for testing or other use-cases. So we should describe what this module accomplishes in a generic sense independently of our roadmap.